dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/production/LksgProductCategory.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.lksg.categories.general.subcategories.production
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the LKSG questionnaire regarding a single product category for the "Product Categories" field
+ */
+data class LksgProductCategory(
```

Comment:
I think this should be `LksgProcurementCategory`.

Reply:
I believe you are right. The Data Dictionary specified something wrong in the field name. But the specified question makes it look like you are right.

Reply:
Yes @SiegfriedSobkowiak  and I noticed this when we read the question in the data dictionary today.

Reply:
Had a call and agreed on changin this

Reply:
Did that everywhere. To be pushed.

Reply:
Description should be changed too. It still references to "products".

Reply:
No more

Reply:
👍